tests/test_e2e.py

```diff
@@ -106,12 +86,9 @@ def test_tensorrtllm_e2e(self, protocol, setup_and_teardown):
         model = os.environ.get("TRTLLM_MODEL")
         assert model is not None, "TRTLLM_MODEL env var must be set!"
         self._import(model, backend="tensorrtllm")
-        pid = utils.run_server()
-        setup_and_teardown.pid = pid
-        utils.wait_for_server_ready()
-
-        self._infer(model, prompt=PROMPT, protocol=protocol)
-        self._profile(model, backend="tensorrtllm")
+        with utils.MockServer() as _:
```

Comment:
Looks like `test_tensorrtllm_e2e` still has `setup_and_teardown` from failing pipeline - double check it's removed everywhere.

Reply:
Just removed it. I don't know how it didn't error out when I ran `pytest`.